src/PKSim.CLI.Core/PKSim.CLI.Core.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Lol, where is the difference between those 2 lines?

Reply:
Can't see any but after I get some conflicts with different core component versions I always need to edit the project files manually so maybe github does not like this... ;)